code-challenges.test.js

```diff
@@ -14,82 +12,45 @@ const { expect } = require("@jest/globals");
 
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+
 // a) Create a test with expect statements for each of the variables provided.
+
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
+
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-// create test 
-// create function that takes in a number 2 < 
-// return array of fibonacci sequence
-describe("fibonacci", () => {
-    it("takes in a number and creates a fibonacci sequence", () => {
-        expect(fibonacci(6)).toEqual(0);
-        expect(fibonacci(10)).toEqual(0, 1,  1,  2,  3, 5, 8, 13, 21, 34, 55);
-    })
-})
```

Comment:
I don't think this output is quite right. Make sure you are taking the exact given input and output.